templates/crawlee_cheerio_typescript/.gitignore

```diff
@@ -0,0 +1,7 @@
+# This file tells Git which files shouldn't be added to source control
+
+.idea
+dist
+node_modules
+apify_storage
+memory_storage
```

Comment:
Why does memory storage need a folder? 😄

Reply:
So users can read the values? 

Under the hood it works similarly to the storage-local, instead of sqlite we store things in memory, but we also dump them to file system (via worker thread, so it does not affect performance much).